src/platform/platform_win.cpp

```diff
@@ -60,12 +60,6 @@
 
 #include <libssh/sftp.h>
 
-// Needed for OpenSSL dll compatibility
-extern "C"
-{
-#include <openssl/applink.c>
-}
-
```

Comment:
How did you come to the conclusion that this could now be removed?

Reply:
It's an old way to including `applink` which isn't compatible with how vcpkg fetches OpenSSL. It's now replaced with `OpenSSL::applink` in `CMakeLists.txt`.